app/controllers/report_controller/schedules.rb

```diff
@@ -397,7 +394,7 @@ def schedule_get_form_vars
       # rebuild hash to hold user's email along with name if user record was found for display, defined as hash so only email id can be sent from form to be deleted from array above
       @email_to = {}
       @edit[:new][:email][:to].each_with_index do |e, _e_idx|
-        u = User.find_by_email(e)
+        u = User.find_by(:email => e)
```

Comment:
you canot change this one:

https://github.com/ManageIQ/manageiq/blob/master/app/models/user.rb#L89

Reply:
Fixed.